docs/source/usage/parameters.rst

```diff
@@ -449,6 +449,11 @@ Lattice Elements
                 openPMD `iteration encoding <https://openpmd-api.readthedocs.io/en/0.14.0/usage/concepts.html#iteration-and-series>`__: (v)ariable based, (f)ile based, (g)roup based (default)
                 variable based is an `experimental feature with ADIOS2 <https://openpmd-api.readthedocs.io/en/0.14.0/backends/adios2.html#experimental-new-adios2-schema>`__.
 
+            * ``<element_name>.cycle_intervals`` (``int``, default value: ``1``)
+
+                For periodic lattice, only output every N cycles (turns).
```

Comment:
Elsewhere, we use the term "periods" instead of "cycles".  For example, we set lattice.periods = N.  I think this terminology is more standard, and it might be good to use this for consistency.

Reply:
Good point, currently based on
https://github.com/ECP-WarpX/impactx/blob/3a8ab8dc981aafccc9ab3de167e222d986f8b374/src/ImpactX.cpp#L183-L187

Reply:
For the user-facing name, what about "period_sample_interval"?

Reply:
That sounds good. I would write as in warpx plural, `period_sample_intervals`, because in a follow-up we will enable users to write multiple intervals here: https://warpx.readthedocs.io/en/latest/usage/parameters.html#intervals-parser